packages/client/src/client.js

```diff
@@ -450,7 +450,7 @@ export class PercyClient {
     this.log.debug(`Verifying comparison tile with sha: ${sha}`);
 
     try {
-      return await this.get(`comparisons/${comparisonId}/tiles/verify`, {
+      return await this.post(`comparisons/${comparisonId}/tiles/verify`, {
```

Comment:
Shouldn't we have a test change related to this ?

Reply:
there is no specific test for the the request made in client.js. There are test written for this function. and there are separate tests written for get and post request.

Reply:
added test